src/components/response_error.tsx

```diff
@@ -8,63 +8,30 @@ import Tile from './tile';
 // utils
 import { I18n } from '../utils/i18n';
 
-/**
- * Map of all API error codes that should be translated.
- * The variables "code", "field_name" and "title" are available here.
- */
-export const ERROR_CODE_TO_I18NC_TEMPLATE = {
-
-    // generic errors
-    "genericError": "There was a problem on the server, please try again. (Code: %{code})",
-    "apiResponseError": "There was a problem on the server, please try again.",
-
-    // generic field errors
-    "fieldError": "%{field_name} was invalid: %{title}.",
-    "lessThanMinimum": "%{field_name} does not have enough characters.",
-
-    // specific field errors
-    "emailInvalid": "Email address was not valid. Please use only lower-case characters and a full domain name.",
-    "fullNameInvalid": "Full name was not valid. Please use only latin characters, \
-        numbers and simple separators (single space, underline or hyphen).",
-    "usernameInvalid": "Username was not valid. Please use only latin characters, \
-        numbers and simple separators (single space, underline or hyphen).",
-    "passwordInvalid": "Password was not valid. Please use at least eight characters, \
-        at least one number and both lower and uppercase letters and special characters.",
-    "badPasswordRepeat": "Repeated password did not match original password.",
-    "dataUriImageInvalid": "Profile image was not valid. Please use a JPEG or PNG image."
-
-}
-
-/**
- * Map of all form field names that should be translated.
- */
-export const FIELD_NAME_TO_I18NC = {
-
-    // user data
-    "username": I18n.translate("form field username", "Username"),
-    "email": I18n.translate("form field email", "Email Address"),
-    "name": I18n.translate("form field name", "Full Name"),
-    "password": I18n.translate("form field password", "Password"),
-    "password_repeat": I18n.translate("form field password_repeat", "Repeat Password"),
-
-}
+export type ErrorMessageTemplates = {[key: string]: string};
```

Comment:
add something like:
// a mapping of error codes to translated error messages which may include the error parameters

and
// a mapping of field names to their translated descriptions

Reply:
Done.